src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,116 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: $card-width;
+
+  font-family: Roboto, sans-serif;
+  border: 1px solid $main-border-color;
+  background-color: $main-background-color;
+  border-radius: 5px;
+  color: $main-text-color;
+  transition: transform 0.3s ease-in-out;
+
+  &__img-link {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-top: 0;
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 0;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+
+    color: $secondary-text-color;
+  }
+
+  &__stars-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    margin-bottom: 24px;
+  }
+
+  &__stars {
+    margin: 0;
+    padding: 0;
+  }
+
+  &__reviews {
+    margin: 0;
+
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+  }
+
+  &__price-container {
+    margin-bottom: 16px;
+
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+
+    color: $secondary-text-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
```

Comment:
u don't need it, because u already have `justify-content: space-between` property on parent tag

Reply:
> u don't need it, because u already have `justify-content: space-between` property on parent tag

Not fixed